app/src/main/java/com/zeapo/pwdstore/git/operation/GitOperation.kt

```diff
@@ -167,9 +149,10 @@ abstract class GitOperation(gitDir: File, internal val callingActivity: Fragment
                     }
                     when (result) {
                         is BiometricAuthenticator.Result.Success -> {
-                            withSshKeyAuthentication(CredentialFinder(callingActivity, authMode)).execute()
+                            registerAuthProviders(
+                                SshAuthData.SshKey(CredentialFinder(callingActivity, AuthMode.SshKey)))
                         }
-                        is BiometricAuthenticator.Result.Cancelled -> callingActivity.finish()
+                        is BiometricAuthenticator.Result.Cancelled -> return Result.success()
```

Comment:
I had to do this because otherwise we were finishing away `PasswordStore` on biometric cancellation.

Reply:
It might be cleaner to throw an `SSHException` with `disconnectReason` `AUTH_CANCELLED_BY_USER` here, but not by much. Your call.

Reply:
I'll make the change.